modal/stub.py

```diff
@@ -422,6 +422,7 @@ def function(
         proxy: Optional[_Proxy] = None,  # Reference to a Modal Proxy to use in front of this function.
         retries: Optional[Union[int, Retries]] = None,  # Number of times to retry each input in case of failure.
         concurrency_limit: Optional[int] = None,  # Limit for max concurrent containers running the function.
+        allow_concurrent_inputs: Optional[int] = None,  # Allow container to handle multiple inputs simultaneously.
```

Comment:
this description makes it sound like it's a boolean – prob worth pointing out what the number describes